root/static/scripts/common/components/EntityLink.js

```diff
@@ -374,6 +374,16 @@ $ReadOnlyArray<Expand2ReactOutput> | Expand2ReactOutput | null => {
           {content}
         </React.Fragment>
       );
+    } else if (entity.entityType === 'release' &&
```

Comment:
Following [the previous comment](https://github.com/metabrainz/musicbrainz-server/pull/3256#issuecomment-2095730463), it seems that this would do:

```suggestion
    };

    if ((entity.entityType === 'release' || entity.entityType === 'release_group') &&
```

* I didn’t test it.
* It might also be worth to reorder conditional blocks more logically.